.github/workflows/add-issues-to-project.yaml

```diff
@@ -0,0 +1,16 @@
+name: Automatically add newly created issues to the backlog project
+
+on: [issues]
+
+jobs:
+  github-actions-automate-projects:
+    runs-on: ubuntu-latest
+    steps:
+    - name: add-new-issues-to-org-level-issue-backlog-project
+      uses: docker://takanabe/github-actions-automate-projects:v0.0.1
+      if: github.event_name == 'issues' && github.event.action == 'opened'
+      env:
+        GITHUB_TOKEN: ${{ secrets.KATA_GITHUB_ACTIONS_TOKEN }}
```

Comment:
I suppose you have created a token here with restricted access rather than using the default GITHUB_TOKEN?

Reply:
That's right - `KATA_GITHUB_ACTIONS_TOKEN` is a new token I've created specifically for (this) action. It's technically a personal access token my user has created on the org and then each repo has a "secret" with the value of this token. This is documented here:

- https://github.com/marketplace/actions/automate-projects#organization-based-project